### ISSUE TITLE
About the PSNR of the reproduced results.

### ISSUE THREAD

**Issue by Wardwarf-Li** [zzs1994/CVQN#3]
Thanks for your great work! But I can't perfectly reproduce the results of the 'predefined' setting in your paper with your code. The PSNR on Kodak should be 28.024 dB but I got 26.949 dB. BTW, the MS-SSIM and BPP are correct.

**Comment by zzs1994** [comment]
[gmm_32_357_202004240112_train.log](https://github.com/zzs1994/CVQN/files/6169898/gmm_32_357_202004240112_train.log)

**Comment by zs-zhong** [comment]
Thanks for your interest! I am not sure what happen to your training process. I uploaded my training log for you. You can check my training setting whether is consistent with yours. Hope it could help!
[gmm_32_357_202004240112_train.log](https://github.com/zzs1994/CVQN/files/6169956/gmm_32_357_202004240112_train.log)

**Comment by Wardwarf-Li** [comment]
Thanks for your reply! I read your training log and retrained the 'predefined' model following the guidance. Unfortunately, I got PSNR at around 26.9dB again.  Besides, I found your training dataset is smaller (124 iterations per epoch, mine is 127 per epoch). My training dataset has 4,035 images in total (2,650 from Flickr2K, 800 from DIV2k and 585 from CLIC20). Could the problem be here?
[gmm_32_357_predefine_202103191432_train.log](https://github.com/zzs1994/CVQN/files/6180121/gmm_32_357_predefine_202103191432_train.log)

**Comment by zs-zhong** [comment]
I just checked my training set: It just contains 3,938 images. I used the training images from Flickr2K, DIV2K, and CLIC'18. However, I revisited the website of CLIC'18 and the download link of the training set seemed disabled. I think the inconsistency in the training set may be the problem.

**Comment by Wardwarf-Li** [comment]
If possible, can you provide me a copy of the CLIC'18 training set? I will be very appreciated for that!!!

**Comment by zs-zhong** [comment]
I am so sorry that I don't have the original CLIC'18 training set now. I have renamed all the images and shuffled them for training.  I also notice your evaluation process is much faster than mine. In my evaluated set, it contains 24 uncompressed images (from the Kodak dataset).

**Comment by Wardwarf-Li** [comment]
I used 24 uncompressed 768x512 images from Kodak dadaset for evaluation. I'm not sure if we used the same Kodak dataset.
